cmd/bucket-handlers.go

```diff
@@ -1546,7 +1546,7 @@ func (api objectAPIHandlers) HeadBucketHandler(w http.ResponseWriter, r *http.Re
 		return
 	}
 
-	if s3Error := checkRequestAuthType(ctx, r, policy.ListBucketAction, bucket, ""); s3Error != ErrNone {
+	if s3Error := checkRequestAuthType(ctx, r, policy.HeadBucketAction, bucket, ""); s3Error != ErrNone {
 		writeErrorResponseHeadersOnly(w, errorCodes.ToAPIErr(s3Error))
 		return
 	}
```

Comment:
We did this on purpose @jiuker to keep things simple on the policy side, such that users don't have to explicitly set `s3:HeadBucket` action.

Since it is sufficient enough to have `ListBucketAction` - because no one really cares for just `HeadBucket`, we can, however, pass both in this scenario as actions.

Reply:
> We did this on purpose @jiuker to keep things simple on the policy side, such that users don't have to explicitly set `s3:HeadBucket` action.
> 
> Since it is sufficient enough to have `ListBucketAction` - because no one really cares for just `HeadBucket`, we can, however, pass both in this scenario as actions.

@harshavardhana You mean checking two permissions at the same time?